# Goal:
string move     # from kociemba solver
---
# Result
bool success
---
# Feedback
